File: 프로그래머스/0/120814. 피자 나눠 먹기 （1）/피자 나눠 먹기 （1）.py
def solution(n):
    if n % 7 == 0 :
        sum = n // 7 
    else :
        a = 0
        sum = 0
    
        while a < n :
            if n // 7 == a:
                sum = a+1
            a += 1
    
    answer = sum
    return answer
    

File: 프로그래머스/0/120814. 피자 나눠 먹기 （1）/피자 나눠 먹기 （1）.java
class Solution {
    public int solution(int n) {
        int answer = 0;
        if(n % 7 == 0){
            answer = n / 7; 
        }else {
            for(int i = 0; i < n; i++){
                if( n / 7 == i){
                    answer = i + 1;   
                }    
            }
        }
        return answer;
    }
}